util/types/src/core/cell.rs

```diff
@@ -165,7 +165,7 @@ impl CellMeta {
 }
 
 /// TODO(doc): @quake
-#[derive(PartialEq, Debug, Eq)]
+#[derive(PartialEq, Debug, Eq, Clone)]
```

Comment:
unrelated change to wasm?

Reply:
https://github.com/driftluo/ckb-light-client/blob/wasm-support-with-indexdb/light-client-lib/src/storage/mod.rs#L159-L164
impl this must have